packages/material-ui-styles/src/withStyles/withStyles.d.ts

```diff
@@ -75,8 +76,9 @@ export interface StyledComponentProps<ClassKey extends string = string> {
 }
 
 export default function withStyles<
-  S extends Styles<any, any>,
-  Options extends WithStylesOptions = {}
+  S extends Styles<Theme, any>,
+  Options extends WithStylesOptions<Theme> = {},
```

Comment:
should this use `ThemeOfStyles` too?

Reply:
Exactly. I could've just pointed out the helper type but there were some issues with `createStyles` too so I'd opted for a quick fix.

Reply:
ok, will fix it